media-libs/opencolorio/opencolorio-2.3.2.ebuild

```diff
@@ -0,0 +1,173 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit cmake python-single-r1 virtualx
+
+DESCRIPTION="Color management framework for visual effects and animation"
+HOMEPAGE="https://opencolorio.org https://github.com/AcademySoftwareFoundation/OpenColorIO"
+SRC_URI="https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/OpenColorIO-${PV}"
+
+LICENSE="BSD"
+SLOT="0/$(ver_cut 1-2)"
+# minizip-ng: ~arm ~arm64 ~ppc64 ~riscv
+# osl: ~riscv
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+CPU_USE=(
+	x86_{avx,avx2,avx512f,f16c,sse2,sse3,sse4_1,sse4_2,ssse3}
+	# requires https://github.com/DLTcollab/sse2neon
+	# arm_neon
+)
+IUSE="apps ${CPU_USE[@]/#/cpu_flags_} doc opengl python test"
+# TODO: drop opengl? It does nothing without building either the apps or the testsuite
+REQUIRED_USE="
+	apps? ( opengl )
+	doc? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	test? ( opengl )
+"
+
+RDEPEND="
+	dev-cpp/pystring
+	>=dev-cpp/yaml-cpp-0.7.0:=
+	dev-libs/expat
+	>=dev-libs/imath-3.1.5:=
+	sys-libs/minizip-ng
+	sys-libs/zlib
+	apps? (
+		media-libs/lcms:2
+		>=media-libs/openexr-3.1.5:=
+	)
+	opengl? (
+		media-libs/freeglut
+		media-libs/glew:=
+		media-libs/libglvnd
+	)
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep 'dev-python/pybind11[${PYTHON_USEDEP}]')
+	)
+"
+DEPEND="${RDEPEND}"
+# TODO: OSL tests would need OIIO, leading to a circular dependency. If OIIO
+# isn't found this test will be skipped (automagic if found?)
+BDEPEND="
+	virtual/pkgconfig
+	doc? (
+		app-text/doxygen
+		$(python_gen_cond_dep '
+			dev-python/breathe[${PYTHON_USEDEP}]
+			dev-python/recommonmark[${PYTHON_USEDEP}]
+			dev-python/six[${PYTHON_USEDEP}]
+			dev-python/sphinx[${PYTHON_USEDEP}]
+			dev-python/sphinx-press-theme[${PYTHON_USEDEP}]
+			dev-python/sphinx-tabs[${PYTHON_USEDEP}]
+			dev-python/testresources[${PYTHON_USEDEP}]
+		')
+	)
+	opengl? (
+		media-libs/freeglut
+		media-libs/glew:=
+		media-libs/libglvnd
+	)
+"
+# 	test? (
+# 		>=media-libs/openimageio-2.2.14
+# 		>=media-libs/osl-1.11
+# 	)
+# "
+
+# Restricting tests, bugs #439790 and #447908
+# upstream doesn't care https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1361
```

Comment:
I don't think this means they don't care, does it? It means they think the failures reported aren't interesting?

We should also be really careful with phrasing with this sort of thing. Upstreams often look at ebuilds. We don't want to assert blame or anything. 

cc @parona-source

Reply:
I agree its overly harsh. They care but its not a priority. M1 Macs started to hit this so it may garner more interest now.

Reply:
Not sure you care when you do string representation comparisons of floating point numbers in tests of a numerical library. But alas.